GOVERNANCE.md

```diff
@@ -1,53 +1,87 @@
-
-**4.4.	Publication and Submission.**  Upon the designation of a Draft Specification as an Approved Specification, the Maintainer(s) will publish the Approved Specification in a manner agreed upon by the Working Group Participants (i.e., Working Group Participant only location, publicly available location, Working Group maintained website, Working Group member website, etc.).  The publication of an Approved Specification in a publicly accessible manner must include the terms under which the Approved Specification is being made available under.
-
-**4.5.	Submissions to Standards Bodies.**  No Draft Specification or Approved Specification may be submitted to another standards development organization without Working Group Approval. Upon reaching Approval, the Maintainer(s) will coordinate the submission of the applicable Draft Specification or Approved Specification to another standards development organization. Working Group Participants that developed that Draft Specification or Approved Specification agree to grant the copyright rights necessary to make those submissions.
-
-## 5. Non-Confidential, Restricted Disclosure.
-
-Information disclosed in connection with any Working Group activity, including but not limited to meetings, Contributions, and submissions, is not confidential, regardless of any markings or statements to the contrary.  Notwithstanding the foregoing, if the Working Group is collaborating via a private repository, the Participants will not make any public disclosures of that information contained in that private repository without the Approval of the Working Group.  
+# FDC3 Governance Policy
+
+This document provides the governance policy for the development of the FDC3 specification and related materials in the FDC3 Standard Working Group (the “Working Group”).
+
+## 1.	Roles.
+
+Working Group contributors include the following roles:
+
+### 1.1. Participants
+
+“Participants” are those that have made Contributions to the Working Group subject to the [Community Specification Contribution Policy 1.0](https://github.com/finos/standards-project-blueprint/blob/master/governance-documents/6._Contributing.md).  The [FDC3 Standard Working Group](https://github.com/finos/FDC3/issues?q=label%3A%22Standard+WG+Meeting%22) has the specific purpose of defining and releasing subsequent updates to the Standard. In practice, that means people that attend and contribute to meetings, raise issues, pull requests (to submit patches to the Standard) and reviews.
+
+#### How do you become a Participant?
+
+Becoming an FDC3 Participant is as easy as attending a meeting and/or raising issues for changes you'd like see in the Standard, commenting on issues others have raised or even asking questions (which can often result in the clarification of the Standard's documentation to help others with the same questions in future).
+
+### 1.2. Voting Participants
```

Comment:
@robmoffat - it seems that "Voting Participants" are different than "Participants", whereas my understanding is that all Participants have the right to vote.

By reading the entire file, I think that what you write makes sense, but we're effectively adding a new Role, which IMO needs some review and approval from the FDC3 (core) maintainer's team.

Reply:
I think we should check with @kriswest, but really this is just clarifying what was already the case I think

Reply:
Rob is correct that it's always been like this - they are not intended to be separate groups, but not every participant gets around to signing up for voting and agreeing to the terms. The sign-up has always been a way to get people to agree to the license terms and get that documented - its not a great process, but with participants that can't do Github, we haven't had a reasonable alternative proposed.

To deal with @maoo's concern we could just tweak the structure here to keep them together. Will propose an edit